### PR TITLE
docs: Add offline deployment document. #396

### LINKED ISSUE #396: [feature] Offline deployment document.
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by runkecheng**
### What type of PR is this?

doc

### Which issue(s) this PR fixes?

Fixes #396 

### What this PR does?

Summary:

- Provide deployment resource compression package `radondb-mysql-resources.tar.gz` in Release.
- Tutorial for offline deployment 

### Special notes for your reviewer?

The resources needed to deploy have been uploaded to Release.

Need to verify that the process can go through.

### CODE REVIEW

**Review by andyli029 — CHANGES_REQUESTED** [review]
deployment-resources.tar.gz->?
v2.1.2-resources->?

@molliezhang  @runkecheng

File: docs/zh-cn/deploy_radondb-mysql_operator_on_k8s_offline.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+### 步骤 1: 准备部署资源
+
+```shell
+## 下载离线部署资源
+wget https://github.com/radondb/radondb-mysql-kubernetes/releases/latest/download/deployment-resources.tar.gz
+
+## 解压
+tar -xvzf deployment-resources.tar.gz
+
+## 导入镜像（需要在每个运行数据库的 worker 上执行）
```
Comment (andyli029) [inline docs/zh-cn/deploy_radondb-mysql_operator_on_k8s_offline.md:46]: cd dir

**Review by molliezhang — COMMENTED** [review]
LGTM

**Comment by qianfen2021** [comment]
LGTM

**Review by andyli029 — APPROVED** [review]
LGTM.

**Pull request merged**